### DISCUSSION TITLE
Vmware: Persistent Storage

### DISCUSSION THREAD

**Discussion by trebonian** [bottlerocket-os/bottlerocket#1632] [Q&A]
What iis(will be) - the supported mechanism for adding persistent storage to vmware variants. Are there any CSI plugins that can be used?

I have already successfully joined a bottlerocket node into a pre-existing vmware cluster, and would like to know how to create Persistent Volume Claims...

**Comment by bcressey** [comment]
We've tested with the vSphere CSI driver so that should work. Please give it a try and let us know if you run into problems.
